packages/seo-integration/src/replacement-variables.js

```diff
@@ -0,0 +1,111 @@
+import { select } from "@wordpress/data";
+import { addFilter, removeFilter } from "@wordpress/hooks";
+import { __ } from "@wordpress/i18n";
+import createReplacementVariables from "@yoast/replacement-variables";
+import { SEO_STORE_NAME } from "@yoast/seo-store";
+import { curryRight, mapValues, noop, reduce, unionBy } from "lodash";
+
+/**
+ * Creates the default replacement variable configurations, for use within the SEO store context.
+ *
+ * There are more replacement variables outside of the SEO store context.
+ * @see [Available replacement variables in Yoast SEO for WordPress]{@link https://yoast.com/help/list-available-snippet-variables-yoast-seo/}
+ *
+ * @returns {Object.<string, ReplacementVariableConfiguration>} The default replacement variable configurations, keyed by name for easy reference.
+ */
+export const createDefaultReplacementVariableConfigurations = () => ( {
+	title: {
```

Comment:
I'd say change `@yoast/replacement-variables` to accept an object also or make this an array. Both fine by me as long as we're consistent!

Reply:
This was an array first, but with the addition of the `analysisType` scope, it makes more sense to have this as an Object so that the implementer can easily pick the ones they want to use in for each type.